3rdParty/fuerte/include/fuerte/message.h

```diff
@@ -193,8 +194,13 @@ class Request final : public Message {
   /// @brief request header
   RequestHeader header;
 
+
   MessageType type() const override { return MessageType::Request; }
   MessageHeader const& messageHeader() const override { return header; }
+  void setFuzzReqHeader(std::string& fuzzHeader) {_fuzzReqHeader = fuzzHeader;}
```

Comment:
Also just wondering here if we need separate APIs to set a boolean for fuzzing.
Can't it be simplified to just
```
// implies setFuzzerReq().
void setFuzzReqHeader(std::string& fuzzHeader) { _fuzzReqHeader = fuzzHeader; }
std::optional<std::string> getFuzzReqHeader() const { return _fuzzReqHeader; }

bool getFuzzerReq() const noexcept { return _fuzzReqHeader.has_value(); }
```

Reply:
Addressed